wandb/sdk/wandb_run.py

```diff
@@ -481,62 +481,52 @@ class RunStatus:
 
 
 class Run:
-    """A unit of computation logged by wandb. Typically, this is an ML experiment.
+    """A unit of computation logged by W&B. Typically, this is an ML experiment.
+
+    Call [`wandb.init()`](https://docs.wandb.ai/ref/python/init/) to create a
+    new run. This will start a new run and return a `wandb.Run` object.
+    Each run is associated with a unique ID, which is used to identify
+    the run in the W&B UI.
+
+    You can log data to that run with `wandb.log()`. There is only ever at
+    most one active `wandb.Run` in any process. Anything you log with
+    `wandb.log()` is sent to that run. See
+    [Create an experiment](https://docs.wandb.ai/guides/track/launch) or
+    [`wandb.init`](https://docs.wandb.ai/ref/python/init/) API reference page
+    or more information.
+
+    In distributed training, you can either create a single run in the rank 0
+    process and then log information only from that process, or you can create
+    a run in each process, logging from each separately, and group the results
+    together with the `group` argument to `wandb.init`. See
+    [Log distributed training experiments](https://docs.wandb.ai/guides/track/log/distributed-training).
```

Comment:
...or use shared mode and log to the same run - I think we should also describe it here.